craco.config.js

```diff
@@ -60,9 +49,25 @@ module.exports = {
     // https://webpack.js.org/configuration
     configure: (webpackConfig) => ({
       ...webpackConfig,
+      // https://github.com/facebook/create-react-app/discussions/11767#discussioncomment-2421668
```

Comment:
I think we might be able in the future to even kill `craco`

Reply:
Yes, but we still need it for:
```
modules: [path.resolve(__dirname, 'src/custom'), ...webpackConfig.resolve.modules],
```

Without that code it doesn't work